views/users/accounts-import-complete.pug

```diff
@@ -0,0 +1,21 @@
+.row
+  .col.s12
+    .card.center
+      .card-image
+        a.btn-floating.halfway-fab.waves-effect.waves-light.red.left#account-import-list-back-button
+          i.material-icons arrow_back
+      .card-content
+        include ../checkmark
+        h5.card-title Import Job Completed
+        h6 The list of students' accounts has been imported
+      .card-action
+        .flow-left
+          h5.alignTextLeft &nbsp;Report:
+          h6.alignTextLeft &nbsp;&nbsp;&nbsp;&nbsp;Total number of students added: #{added}
```

Comment:
anyone knows a better way to do this?
I want to do
Reports
    stuff
    stuff

Reply:
thought wee were gunna make a space generator in `common.js`

Reply:
Thats how I would do it.